packages/utils/src/commonwell/migrate-patients.ts

```diff
@@ -0,0 +1,177 @@
+            () =>
+              axios.delete(resetUrl, {
+                params: {
+                  cxId,
+                  source: MedicalDataSource.COMMONWELL,
+                },
+              }),
+            {
+              maxAttempts: NETWORK_RETRY_MAX_ATTEMPTS,
+              initialDelay: NETWORK_RETRY_INITIAL_DELAY_MS,
+              retryOnTimeout: NETWORK_RETRY_ON_TIMEOUT,
+            }
+          );
+
+          log("Running patient discovery...");
+          const pdUrl = `${apiUrl}/internal/commonwell/patient-discovery/${patientId}`;
+          const pdParams = new URLSearchParams({
+            cxId,
+            rerunPdOnNewDemographics: rerunPdOnNewDemographics.toString(),
+          });
+          const resp = (await executeWithNetworkRetries(() => axios.post(`${pdUrl}?${pdParams}`), {
+            maxAttempts: NETWORK_RETRY_MAX_ATTEMPTS,
+            initialDelay: NETWORK_RETRY_INITIAL_DELAY_MS,
+            retryOnTimeout: NETWORK_RETRY_ON_TIMEOUT,
+          })) as PdResponse;
+          log(`Request ID - ${JSON.stringify(resp.data.requestId)}`);
+
+          // 3. Delete patient in CommonWell
+          log("Deleting patient in CommonWell...");
+          const queryMeta = organizationQueryMeta(cwOrgName, {
+            npi: addOidPrefix(oldCwOid),
+          });
+
+          const cwPatientId = buildCWPatientId(oldCwOid, patientId);
+
+          await executeWithNetworkRetries(() => commonWell.deletePatient(queryMeta, cwPatientId), {
+            maxAttempts: NETWORK_RETRY_MAX_ATTEMPTS,
+            initialDelay: NETWORK_RETRY_INITIAL_DELAY_MS,
+            retryOnTimeout: NETWORK_RETRY_ON_TIMEOUT,
+          });
+
+          log("Patient migration completed successfully");
+          successCount++;
+        } catch (err) {
+          log(`Failed to migrate patient: ${err}`);
+          failures[patientId] = String(err);
+          failureCount++;
+        }
+      }
+
+      if (i < patientChunks.length - 1) {
+        const sleepTime = getDelayTime({
+          log,
+          minimumDelayTime,
+          defaultDelayTime: delayTime,
+        });
+        console.log(`Chunk ${i + 1} finished. Sleeping for ${sleepTime} ms...`);
+        await sleep(sleepTime);
+      }
+    }
+
+    console.log(`\nMigration summary:`);
+    console.log(`Successfully migrated: ${successCount} patients`);
+    console.log(`Failed to migrate: ${failureCount} patients`);
+
+    if (failureCount > 0) {
+      console.log("Failed patients:");
+      Object.entries(failures).forEach(([patientId, error]) => {
+        console.log(`- Patient ${patientId}: ${error}`);
+      });
+    }
+  } catch (err) {
+    const msg = "Patient migration failed.";
+    console.log(`${msg}. Error - ${err}`);
+  }
+}
+
+async function displayInitialWarningAndConfirmation(numberPatients: number) {
+  console.log("\n\x1b[31m%s\x1b[0m\n", "---- ATTENTION - THIS IS NOT A SIMULATED RUN ----"); // https://stackoverflow.com/a/41407246/2099911
```

Comment:
nit: prefer not to use links to 3rd parties to explain what code means - can just say the first portion of this is to set the color if needed